Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -107,6 +107,50 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
             }
         }
 
+        [Theory]
+        [MemberData("ResponseModelTypes")]
```

Comment:
I went for consistency here. not knowing what versions of VS contributors use, even if I guess it's safe they use a recent enough version. The .NET Core port should be merged in the near future, and since the new .csproj involves using VS 2017+, these updates will be totally safe.